### PR TITLE
fail tests when go fmt fails

### PR DESCRIPTION
**Pull request by grosser**
@williammartin

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,10 +7,16 @@ go:
...
+  - go get -v ./...
+  - go build ./...
   - go get github.com/onsi/ginkgo
   - go install github.com/onsi/ginkgo/ginkgo
 
-script: env GO111MODULE=on $HOME/gopath/bin/ginkgo -p -r --randomizeAllSpecs --failOnPending --randomizeSuites --race && env GO111MODULE=on go vet
+script: |
+  $HOME/gopath/bin/ginkgo -p -r --randomizeAllSpecs --failOnPending --randomizeSuites --race &&
+  go vet &&
+  [ -z "`gofmt -l -e -s -w .`" ]
```
Comment (grosser) [inline .travis.yml]: -l -> list changed files
-e -> all errors
-s -> simplify
-w -> rewrite (so users can run it locally and don't have to bother with the diff)
. -> recursive

**Comment by williammartin** [comment]
If I understand correctly, this PR provides the following functionality:

* Move GO111MODULE into more suitable env key in Travis
* Fail the build if gofmt detects any changes required
* Make changes to satisfy gofmt right now

Is that correct? Can we split this into a few commits?

**Comment by grosser** [comment]
done

**Comment by williammartin** [comment]
Money. Let's see Travis pass. Thanks

**Comment by grosser** [comment]
🍏 📗 💚

**Pull request merged**